tools/bouncer_worker/src/lib/elastic.js

```diff
@@ -65,7 +65,7 @@ const establishIndices = async (client) => Promise.all(
 );
 
 const createElasticClient = async () => {
-	if (!elastic) return undefined;
+	if (!elastic || !elastic.enabled) return undefined;
```

Comment:
could just do 
```suggestion
	if (!elastic?.enabled) return undefined;
```

Reply:
actually i just checked with @Charence, there is no enabled in the config either so this will break the deployments. 

If you don't have elastic configured you shoudln't have an elastic object in the config, is there any reason why you had to do this?

Reply:
@carmenfan, no, well yes in the sense my worker wasn't working but I took another look and i was trying to disable elastic the wrong way! I have reverted the change and it works with the correct config.